workflows/misc_file_types/netCDF_chunk.Rmd

```diff
@@ -12,6 +12,12 @@ atts[1:10,]
 
 ![](https://raw.githubusercontent.com/NCEAS/datateam-training/master/images/netCDF_full_att_preview.png)
 
```

Comment:
I would just add that not all netcdf files will have this information.

Reply:
Does this information here refer to the picture with attributes table?

Reply:
yes